lib/logstash/filters/translate.rb

```diff
@@ -157,39 +207,17 @@ def filter(event)
         @logger.info("refreshing dictionary file")
       end
     end
+
+    #if destination is not set default to #{@field}_translation
+    @destination ||= "#{@field}_translation"
```

Comment:
this needs to be done only once, so I suggest moving this line to the register method.